Documentation language and formatting cleanup (#946)
microsoft/AzureTRE: docs/tre-admins/troubleshooting-guide.md

Received deployment status update message with correlation ID 28fcf096-6962-498b-9d2e-16d38dc6b7f6: {'id': '28fcf096-6962-498b-9d2e-16d38dc6b7f6', 'status': 'deployed', 'message': 'cse-msr-dev-b7f6: Workspace was deployed successfully...'}
```

It should also be evident from the message flow where the current processing is stuck or failed. Failed deployment status should also be available in the `GET /api/workspaces/{workspace_id}` and this is just another way to confirm it.

> ## Checking the Virtual Machine Scale Set(VMSS) instance running resource processor

Reviewer: ```suggestion
## Checking the Virtual Machine Scale Set (VMSS) instance running resource processor
```

If you see messages hanging in the service bus queue then the resource processor is not up and running.

Verify that the VMSS instance is up and healthy.

![VMSS Running](../assets/vmss_running.png)

> The processor runs in a vnet, and you cannot connect to it directly.

Reviewer: ```suggestion
The processor runs in a VNET, and you cannot connect to it directly.
```

1. Connect to the instance using Bastion. Bastion is already deployed, and you can use the username `adminuser`. The password is stored in the keyvault under the secret `resource-processor-vmss-password`

  !!! info

>       You cannot see secrets unless you are added to a suitable Access Policy for the keyvault.

Reviewer: ```suggestion
      You cannot see secrets unless you are added to a suitable access policy for the Key Vault.
```

  ![VMSS Password](../assets/vmss_password.png)

  ![Bastion](../assets/bastion.png "Bastion")

> 1. After logging in you should check the status of **cloud-init** which is used to bootstrap the machine with docker and start the processor. Log files for cloud init are

Reviewer: ```suggestion
1. After logging in you should check the status of **cloud-init** which is used to bootstrap the machine with docker and start the processor. Log files for cloud init are:
```

   - `/var/log/cloud-init.log`
   - `/var/log/cloud-init-output.log`

>    If the docker container is pulled as shown in logs then the resource processor should start.

Reviewer: ```suggestion
   If the Docker container is pulled as shown in logs then the resource processor should start.
```

1. Check the status of the container using `docker ps`

  If you see nothing (and the container was pulled) then the processor has either not started yet or it has crashed.

> 1. Check the status of all docker processes using `docker ps -a` which should show you if the container terminated prematurely.

Reviewer: ```suggestion
1. Check the status of all Docker processes using `docker ps -a` which should show you if the container terminated prematurely.
```
